toolkit-test-app/src/main/java/com/esri/arcgisruntime/toolkit/test/bookmark/map/MapViewModel.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.arcgisruntime.toolkit.test.bookmark.map
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Bookmark
+import com.esri.arcgisruntime.toolkit.test.bookmark.BookmarksRepository
+
+class MapViewModel(map: ArcGISMap) : ViewModel() {
+
+    private val _mapData: MutableLiveData<ArcGISMap> = MutableLiveData(map)
+    val mapData: LiveData<ArcGISMap> = _mapData
```

Comment:
This isn't being used. I think you should be observing this and setting it on the `MapView` opposed to what you are doing here:

https://github.com/Esri/arcgis-runtime-toolkit-android/pull/146/files#diff-85eb21d63939d7e4be2fbe13c15ec13eR47

Reply:
updated